dashboards/solver-rewards-accounting/period-totals.sql

```diff
@@ -20,22 +20,25 @@ realized_fees as (
             ' to ',
             to_char('{{EndTime}}'::timestamptz - interval '1 day', 'YYYY-MM-DD')
         ) as accounting_period,
-        sum(atoms_bought) / 10^18 as realized_fees_eth
-    from gnosis_protocol_v2."trades"
-    where trader in (
+        sum("buyAmount") / 10^18 as realized_fees_eth
+    from gnosis_protocol_v2."GPv2Settlement_evt_Trade"
+    where owner in (
```

Comment:
This change didn't really need to happen here, but I think its better that we rely directly on the event data whenever possible (instead of the abstraction table).